src_ext/Makefile

```diff
@@ -84,6 +87,16 @@ distclean:
 
 LIB_EXTS = extlib re cmdliner graph cudf dose_common dose_debian dose_algo uutf jsonm
 
+# add the preprocessor directive for OCamlMakefile
+CPPO = cppo/cppo
+addcppo = for ml in $1/*.ml $1/*.mli; do echo '(*pp $(CPPO) *)' | cat - $$ml > temp && mv temp $$ml; done
+
+addcppo:
+	$(MAKE) -C cppo all
+	$(call addcppo,dose/common)
```

Comment:
We can get rid of all shell indirections with a short sed command, and I think this will reduce incompatibilities

```
sed -i "1i(*pp $(CPPO) *)" dose/common/*.ml dose/common/*.mli dose/deb/*.ml dose/deb/*.mli dose/algo/*.ml dose/algo/*.mli
```

Also, the target should probably use a stamp file, not to re-add the header every time. I would have it a prerequisite of the `build` target rather than only in the `lib-ext` target, too.